README.md

```diff
@@ -195,14 +195,19 @@ extraneous resources or waiting for a build to complete. The testing is supporte
 Similar to building locally, to test locally, you’ll need access to a personal/team AWS account. To test out:
 
 1. Either on an EC2 instance with the deep-learning-containers repo cloned, or on your local machine, make sure you have
-the images you want to test locally (likely need to pull them from ECR)
-2. In a shell, export environment variable DLC_IMAGES to be a space separated list of ECR uris to be tested. Also set 
+the images you want to test locally (likely need to pull them from ECR). Then change directory into the cloned folder.
+    ```shell script
+    cd deep-learning-containers/
+    ```
+3. In a shell, export environment variable DLC_IMAGES to be a space separated list of ECR uris to be tested. Set 
```

Comment:
Is the extra Set at the end of this sentence a duplicate?

Reply:
Just read through, it's meant to be there. The full sentence reads "Set CODEBUILD_RESOLVED_SOURCE_VERSION to some unique...".

Reply:
Thanks @kace, yes, the "Set" at the end of the line belongs to the next sentence, and was continued in the next line due to formatting and line-character-limit reasons. I have double checked that it comes out as a single paragraph in the rendered output from the markdown text.